### DISCUSSION TITLE
an error while using the latest version of shadcn-ui with pnpm

### DISCUSSION THREAD

**Discussion by Bouts2019** [shadcn-ui/ui#4674] [General]
i followed the official guide to create a shadcn-ui project using pnpm as usual, but when i tried to run 'pnpm dlx shadcn-ui@latest init' this time, it showed me the error ' shadcn-ui is not recognized as an internal or external command'. using the old version with 'pnpm dlx shadcn-ui@0.8.0 init' works fine

**Comment by AdamPilak** [comment]
I have the same issue. There might be a problem with the 0.9.0 version. Switching to shadcn-ui@0.8.0 and then running 'npx shadcn-ui@0.8.0 init' also works fine for me.

**Comment by afganhalbana** [comment]
@AdamPilak awesome your solution works for me, but i wonder what happened to the latest version both `0.9.0` nor `0.9.1` didn't work
